.github/workflows/ci.yml

```diff
@@ -80,24 +80,13 @@ jobs:
       with:
         fetch-depth: 5
     - name: "Set up Ruby ${{ matrix.ruby_version }}"
-      uses: actions/setup-ruby@v1
+      uses: ruby/setup-ruby@v1
       with:
         ruby-version: ${{ matrix.ruby_version }}
-    - name: Cache dependencies
-      uses: actions/cache@v2.1.3
-      with:
-        path: vendor/bundle
-        key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
-        restore-keys: |
-          ${{ runner.os }}-gems-
-    - name: 'Update Rubygems and Bundler'
-      run: |
-        gem update --system --no-document
-        gem update bundler --no-document
-    - name: Set up bundle
-      run: |
-        bundle config path vendor/bundle
-        bundle install --jobs 4 --retry 3
+        bundler-cache: true
+      env:
```

Comment:
For this and the other  config, the docs recommend setting env vars at the job level rather than the step level.

https://github.com/ruby/setup-ruby#bundle-config

I don't know enough about the action to know if it makes a difference, but if i am uncertain I prefer to follow the instructions.

Reply:
Oh nevermind. I think it is just the action author doesn't know you can set the values at the step level.